packages/animations/test/open_container_test.dart

```diff
@@ -1448,6 +1449,75 @@ void main() {
     expect(find.text('Closed 2'), findsNothing);
     expect(find.text('Open 2'), findsOneWidget);
   });
+
+  Widget _createRootNavigatorTest({
+    @required Key appKey,
+    @required Key nestedNavigatorKey,
+    @required bool useRootNavigator,
+  }) =>
```

Comment:
Instead of using `=>` here and throughout this method can you use the regular `{}` syntax with a return statement? I think that will make this block a little more readable.